.ahoy.yml

```diff
@@ -39,7 +39,7 @@ commands:
       ahoy title "Building and starting Docker containers"
       sh bin/docker-compose.sh up -d "$@"
       echo "Initialising database schema"
-      ahoy cli '$APP_DIR/bin/init.sh'
+      ahoy cli '$APP_DIR/scripts/init.sh'
```

Comment:
'bin' is correct.

Reply:
Oh, sorry. If the scripts are still in that location then yeah, `scripts` is the way to go.